app/data-sources/rural-payments/RuralPayments.js

```diff
@@ -6,6 +6,7 @@ import { HttpError } from '../../errors/graphql.js'
 import {
   APIM_ACCESS_TOKEN_REQUEST_001,
   APIM_APIM_REQUEST_001,
+  RURALPAYMENTS_API_ERROR_001,
```

Comment:
extra import not needed, can this type be removed??

Reply:
this should have been removed by the linting.... I'll have to debug that flow another time